src/Bicep.Core/LanguageConstants.cs

```diff
@@ -21,6 +21,8 @@ public static class LanguageConstants
         public const string OutputKeyword = "output";
         public const string VariableKeyword = "var";
         public const string ResourceKeyword = "resource";
+		
+		public static ImmutableSortedSet<string> DeclarationKeywords = new[] {ParameterKeyword, VariableKeyword, ResourceKeyword, OutputKeyword}.ToImmutableSortedSet(StringComparer.Ordinal);
```

Comment:
[nit] tabs 👎

Reply:
Oh I used beyond compare to manually merge this, which must have added tabs /sigh.

Reply:
Untabbed the tabs